src/main/java/com/ait/lienzo/client/widget/LienzoHandlerManager.java

```diff
@@ -193,8 +193,6 @@ public void onClick(final ClickEvent event)
                 m_mouse_button_middle = (event.getNativeButton() == NativeEvent.BUTTON_MIDDLE);
                 m_mouse_button_right = (event.getNativeButton() == NativeEvent.BUTTON_RIGHT);
 
-                event.preventDefault();
```

Comment:
@manstis are you aware of some implications about removing these calls to `prevenDefault`?? 
FYI - It seems those are causing issues with the html events on other components in the page, like the forms. From @pefernan's feedback, as the native event is not firing, another element in the same page but other component. for example, an input the forms, does not loose the focus...